config/abilities/provider_any.rb

```diff
@@ -17,17 +19,15 @@
     cannot :update_permissions, User, &:admin?
 
     # Can't destroy or update role of himself.
-    cannot [:destroy, :update_role], user
+    cannot %i[destroy update_role], user
 
     # Services
-    can [:read, :show, :edit, :update], Service, user.accessible_services.where_values_hash
+    can %i[read show edit update], Service, user.accessible_services.where_values_hash
```

Comment:
> I suggest we don't update unrelated lines just for style

I'd argue that, since we have our code linters "enforced" by Code Climate, we should walk in the direction of having more standardised code. The changes in this file, for example, came from Code Climate complaints. I agree this should be discussed with the rest of the team.

> Also it is not easily understandable for me what actually was the fix.

Do you mean in this line 25? If so, this is about the [Style/SymbolArray cop](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Style/SymbolArray).

Reply:
> Also it is not easily understandable for me what actually was the fix.

I think @akostadinov means (at least this is my opinion also) that when looking at all of the PR changes and there are a lot of style fixes, it's not easy to see what changes are the actual fix, and what are just style updates.

But I like what @thalesmiguel did, and I do it myself - split these change sets in different commits, although it will only be useful if the PR is merged with merge commit and not "squash and merge".

I've added it as an item for discussion in [our workflow doc](https://docs.google.com/document/d/1dNlcUVV2ozZSpnDXgtL5LTu4uRvQ5ZX_xuNVszPfmqc/edit#heading=h.nfaon3vyid9p), we can chat about it later :wink:

Reply:
> although it will only be useful if the PR is merged with merge commit and not "squash and merge".

If it's one of my PRs, you can be sure I'll prever to squash and merge things 😬 

> I've added it as an item for discussion in [our workflow doc](https://docs.google.com/document/d/1dNlcUVV2ozZSpnDXgtL5LTu4uRvQ5ZX_xuNVszPfmqc/edit#heading=h.nfaon3vyid9p), we can chat about it later 😉

YOU. ARE. THE. BEST!

Reply:
I think we do have to do these styling changes at the same time BUT not mixed within the same commit as the fix whatever. To me this PR only lacks 1 proper rebase:
- commit 1: fixes X prblem
- commit 2: 🚨 fixes rubocop warnings

Reply:
I'd like to further discuss this "messes up the history" point outside of this PR. I really don't see it as a shortcoming when compared with the benefits of a standardised code and I'd also argue that the history doesn't really get "messed up". I'd like to understand better the problems that happened here in the past so I can better grasp this issue.
I added a topic to [our workflow doc](https://docs.google.com/document/d/1dNlcUVV2ozZSpnDXgtL5LTu4uRvQ5ZX_xuNVszPfmqc/edit#heading=h.nfaon3vyid9p).